<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CentralWidget</class>
 <widget class="QWidget" name="CentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="categoryList"/>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0">
       <item>
        <widget class="QLineEdit" name="filterPattern">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#aa0000;&quot;&gt;^&lt;/span&gt;pattern&amp;quot; - match begin.&lt;/p&gt;&lt;p&gt;&amp;quot;pattern&lt;span style=&quot; font-weight:600; color:#aa0000;&quot;&gt;$&lt;/span&gt;&amp;quot; - match end.&lt;/p&gt;&lt;p&gt;&amp;quot;&lt;span style=&quot; font-weight:600; color:#aa0000;&quot;&gt;^&lt;/span&gt;pattern&lt;span style=&quot; font-weight:600; color:#aa0000;&quot;&gt;$&lt;/span&gt;&amp;quot; - match entire text.&lt;/p&gt;&lt;p&gt;&amp;quot;&lt;span style=&quot; font-weight:600; color:#aa0000;&quot;&gt;!&lt;/span&gt;pattern&amp;quot; - match case-sensivity.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="messageList">
         <property name="indentation">
          <number>8</number>
         </property>
         <property name="itemsExpandable">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>120</number>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="TagsWidget" name="tagsWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagsWidget</class>
   <extends>QWidget</extends>
   <header>TagsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
